content/rosa/rosa-hcp-portworx/index.md

```diff
@@ -0,0 +1,249 @@
+---
+date: '2024-04-20'
+title: Install Portworx on Red Hat OpenShift Service on AWS (ROSA) with hosted control planes (HCP)
+tags: ["ROSA-HCP", "Portworx", "ROSA"]
+authors:
+  - Nerav Doshi
+---
+
+Portworx storage is a built-for-Kubernetes service that offers flexible and scalable persistent storage for applications in production. In this tutorial we will look at installing Portworx Enterprise on ROSA-HCP. 
+
+## Prerequisites
+You must have a Red Hat OpenShift Service on AWS (ROSA) with [hosted control plane cluster](https://docs.openshift.com/rosa/rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.html#rosa-sts-overview-of-the-default-cluster-specifications_rosa-hcp-sts-creating-a-cluster-quickly) 
+Your cluster must be running OpenShift 4 or higher
+Your cluster must meet AWS prerequisites for ROSA-HCP
```

Comment:
also redundant, having a cluster likely means they meet the pre-reqs.

Reply:
I will remove the pre-req